src/main/java/lotto/model/lotto/LottoRankAward.java

```diff
@@ -0,0 +1,41 @@
+package lotto.model.lotto;
+
+public enum LottoRankAward {
+
+    FIFTH_RANK(3, 5_000, false),
+    FORTH_RANK(4,  50_000, false),
+    THIRD_RANK(5,  1_500_000,false),
+    SECOND_RANK(5, 30_000_000, true),
+    FIRST_RANK(6, 2_000_000_000, false);
+
+    private final int matchedNumberCount;
+    private final int prizeAmount;
+    private final boolean isBonusNumberMatched;
+
+    LottoRankAward(int matchedNumberCount, int prizeAmount, boolean isBonusNumberMatched) {
+        this.matchedNumberCount = matchedNumberCount;
+        this.prizeAmount = prizeAmount;
+        this.isBonusNumberMatched = isBonusNumberMatched;
+    }
+
+    public int getMatchLottoNumberCount() {
+        return matchedNumberCount;
+    }
+
+    public int getWinningMoneyPrize() {
+        return prizeAmount;
+    }
+
+    public boolean getIsBonusMatched() {
```

Comment:
```suggestion
    public boolean isBonusMatched() {
```
Boolean 네이밍 컨벤션에 대해 참고할 만한 글이 있어 남겨봅니다!
https://soojin.ro/blog/naming-boolean-variables

Reply:
와 네이밍에 대해 고민이 정말 많았었는데 좋은 자료 공유해주셔서 감사합니다!!
`getIsBonusMatched` 이 메서드명은 '보너스 번호의 일치여부 값을 얻는다'라는 의미로 get을 사용하였고, 다른 메서드들과 메서드명을 통일해야겠다는 생각에 이렇게 작성했었는데 컨벤션에는 맞지 않았던 것 같네요..!
네이밍이 참 어려운 것 같아요ㅎㅎ
자료 참고해서 더 좋은 네이밍할 수 있도록 해야겠네요 감사합니다~